omniscidb/QueryEngine/RowFuncBuilder.cpp

```diff
@@ -814,6 +814,10 @@ llvm::Value* RowFuncBuilder::convertNullIfAny(const hdk::ir::Type* arg_type,
   if (agg_type->isArray()) {
     CHECK(agg_info.agg_kind == hdk::ir::AggType::kTopK);
     agg_type = agg_type->as<hdk::ir::ArrayBaseType>()->elemType();
+  } else if (agg_info.agg_kind == hdk::ir::AggType::kQuantile) {
+    // Quantile collects orginal values and we don't care about the
+    // result type at the moment.
+    agg_type = arg_type;
```

Comment:
I added some additional information to make it more clear. We don't care about the return type because it is used only when the aggregate is finalized which is not happening in a kernel.

Reply:
Thanks!